### PR TITLE
docs: update documentation

### PR DESCRIPTION
**Pull request by positiveblue**
I will add more documentation about supporting account auto renewal and client side account recovery after the changes for the client side recovery have been merged and we have a new tagged version.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -68,7 +72,7 @@ No. The channel behaves like a regular channel.
 
 ### What happens if the seller force closes a channel before the maturity date?
 
-If the lender force closes they will be banned from the market. This will not be possible in the future.
+Script level enforcement (WIP) will lock the maker's funds in the channel for the lease period. By now, we use a feature in `lnd` to prevent channels from being _cooperatively closed_ by the maker until the expiry height \(what we call the `thaw_height`\).
```
Comment (guggero) [inline docs/faq.md:75]: Does the WIP refer to this sentence or to the script level enforcement in general?

**Review by guggero — APPROVED** [review]
Nice, LGTM :tada:

File: docs/faq.md (hunk 1 above)
Comment (positiveblue) [reply docs/faq.md:75]: Yes, we do not have a way to enforce it yet right?

Comment (guggero) [reply docs/faq.md:75]: The PR was merged and deployed, but you need to actively opt-in into script-enforced leases (if you're running `lnd v0.14.x`). So it's not WIP anymore, just not the default yet.

**Pull request merged**